controller/search.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
 const _ = require('lodash');
+const util = require('util');
```

Comment:
Don't forget to clean up require statements that are no longer used :)

Reply:
@orangejulius Done!